test/unit/cr/CredentialRequest.test.js

```diff
@@ -246,6 +258,6 @@ describe('CredentialRequest', () => {
     const credentialObj = JSON.parse(JSON.stringify(crTest.createCredential()));
 
     const shouldFail = crTest.anchorCredential(credentialObj);
-    return expect(shouldFail).to.be.rejectedWith(Error);
+    return expect(shouldFail).to.eventually.be.rejectedWith(Error);
```

Comment:
Why eventually?

Reply:
Thought it was needed.... learnt something new. Thanks - I removed it